code/modules/mob/living/carbon/xenomorph/castes/carrier/abilities_carrier.dm

```diff
@@ -152,6 +152,11 @@
 	if(!T.check_alien_construction(owner, silent))
 		return FALSE
 
+	if(locate(/obj/effect/alien/weeds/node) in T)

```

Comment:
What about this one for `check_alien_construction()`?

Reply:
you want to be able to build resin walls on top of nodes.